src/libs/actions/Session/index.ts

```diff
@@ -460,6 +461,32 @@ function signUpUser() {
     API.write(WRITE_COMMANDS.SIGN_UP_USER, params, {optimisticData, successData, failureData});
 }
 
+function signInAfterTransitionFromOldDot(transitionURL: string) {
+    const [route, queryParams] = transitionURL.split('?');
+
+    const {email, authToken, accountID, autoGeneratedLogin, autoGeneratedPassword, clearOnyxOnStart} = Object.fromEntries(
+        queryParams.split('&').map((param) => {
+            const [key, value] = param.split('=');
+            return [key, value];
+        }),
+    );
+
+    const setSessionDataAndOpenApp = () => {
+        Onyx.multiSet({
+            [ONYXKEYS.SESSION]: {email, authToken, accountID: Number(accountID)},
+            [ONYXKEYS.CREDENTIALS]: {autoGeneratedLogin, autoGeneratedPassword},
+        }).then(App.openApp);
```

Comment:
Hey folks👋
This has caused an issue in https://github.com/Expensify/App/issues/49964

Explanation from https://github.com/Expensify/App/pull/52694 

> The problem was caused by openApp being called each time ND was opened in hybrid app. It downloads all information needed to open app for the first time. On subsequent start ups Onyx is filled with the data so reconnectApp should be called.